internal/pkg/service/buffer/api/dependencies/dependencies.go

```diff
@@ -200,6 +207,35 @@ func (v *forPublicRequest) RequestID() string {
 	return v.requestID
 }
 
+func (v *forPublicRequest) RequestHeader() http.Header {
+	return v.request.Header.Clone()
+}
+
+func (v *forPublicRequest) RequestClientIP() net.IP {
```

Comment:
Mohlo by se to nějak sjednotit s https://github.com/keboola/keboola-as-code/blob/d47d152b887c0d2cc3071df076cd13895fc01777/internal/pkg/service/buffer/api/http/middleware.go#L142-L153 a https://github.com/keboola/keboola-as-code/blob/d47d152b887c0d2cc3071df076cd13895fc01777/internal/pkg/service/templates/api/http/middleware.go#L135-L146
?

Reply:
jj dik, to som si nevsimol, ze tam uz mame, idem to fixnut

Reply:
Jsem na to zrovna včera narazil, když jsem přemýšlel co s tou IP adresou. 🙂